deploy-board/deploy_board/webapp/helpers/private_builds_helper.py

```diff
@@ -1,99 +0,0 @@
-# Copyright 2016 Pinterest, Inc.
```

Comment:
if this is removed then I think deploy-board/deploy_board/webapp/helpers/s3_helper.py should also be removed